src/index.html

```diff
@@ -9,6 +9,8 @@
     <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Colored blocks</h1>
+    <div class="red"></div>
```

Comment:
add a class `block` to each `div`, in this class you will keep repeating styles in tag `div`